Refine content from first upload (#68)
alphagov/govuk-brand-guidelines: src/graphic-device/dot-use-examples/index.md

Dot behaviours provide flexibility in how thumbnails can be created for a variety of content types, across a scale from informing to inspiring.

Thumbnails should always use the dot in a purposeful way. 

> See these indicative examples of dot use within thumbnails. This example is shown for illustrative purposes only.

Author: Yes you're right! 

```suggestion
See these indicative examples of dot use within thumbnails. These examples are shown for illustrative purposes only.
```

![TODO](./video-thumb-get-help.png)

60 second guides

![TODO](./video-thumb-60s.png)

Step-by-step guides

![TODO](./video-thumb-step.png)

Influencer or presenter

![TODO](./video-thumb-influencer.png)

## Static dot examples

The dot can also be used as a static graphic, adding emphasis and helping guide users through content.

> See these indicative examples of static dot graphics. This example is shown for illustrative purposes only.

Reviewer: Same here, mix of plural "these examples" and singular "this example".

Author: ```suggestion
See these indicative examples of static dot graphics. These examples are shown for illustrative purposes only.
```
